datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/timeline/mappers/SchemaBlameMapper.java

```diff
@@ -62,9 +65,13 @@ public static GetSchemaBlameResult map(List<ChangeTransaction> changeTransaction
         .map(Pair::getSecond)
         .collect(Collectors.toList());
 
-    String selectedSemanticVersion = truncateSemanticVersion(reversedChangeTransactions.get(0).getSemVer());
-    long selectedSemanticVersionTimestamp = reversedChangeTransactions.get(0).getTimestamp();
-    String selectedVersionStamp = reversedChangeTransactions.get(0).getVersionStamp();
+    if (reversedChangeTransactions.isEmpty()) {
```

Comment:
So here is the change- ok nice

Reply:
Yup this is the important part. Made a bunch of things final just to improve readability